docker/README.md

```diff
@@ -3,16 +3,15 @@ Quickstart for localhost development
 
 Install [docker](https://docs.docker.com/engine/installation/) and [docker-compose](https://docs.docker.com/compose/install/).
 
-Following steps will bring up the docker container running Temporal server
-along with all its dependencies (cassandra, statsd, graphite). Exposes Temporal
-frontend on port `7233` and grafana metrics frontend on port `8080`.
+The following steps will bring up a docker container that is running the Temporal server
+and its dependencies (cassandra). The Temporal
+frontend is exposed on port `7233`.
```

Comment:
Probably a question for a another change -- it might be worthwhile to mention bringing up a web browser and navigating to Temporal web page running in the container. http://localhost:8080 (I am current on my phone can’t double check this, @feedmeapples will have better / more reliable data; ).

Reply:
It does mention that in the next section.
I wanted to keep this change focused on updating dependency info - Looks like I will be aiming to reformat this README information in another change.